manifests/config/apache.pp

```diff
@@ -83,6 +83,9 @@
   Boolean $ipa_authentication = $::foreman::ipa_authentication,
   Hash[String, Any] $http_vhost_options = {},
   Hash[String, Any] $https_vhost_options = {},
+  Boolean $keycloak = $foreman::keycloak,
+  String[1] $keycloak_app_name = $foreman::keycloak_app_name,
+  String[1] $keycloak_realm = $foreman::keycloak_realm,
```

Comment:
Do these need to be `$::` prefixed?

Reply:
Technically no and I'd also like to remove that prefix everywhere else. Just haven't gotten around to it. It was needed to make Puppet 3 and Puppet 4 behave in the same way but luckily those dark days are behind us. In this case I'm going to add them to stay consistent with the rest of the file.